Sources/Commands/TestingSupport.swift

```diff
@@ -0,0 +1,143 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Basics
+import SPMBuildCore
+import TSCBasic
+import Workspace
+
+
+/// Internal helper functionality for the SwiftTestTool command and for the
```

Comment:
This code is not new but is moved out from SwiftTestTool into a separate file where it can be used by both SwiftTestTool and SwiftPackageTool.